pages/Identity/Registry.md

```diff
@@ -79,6 +85,10 @@ To retrieve historical values, perform a log search using the `DSNPRegistryUpdat
 The `IRegistry.resolveHandleToId` method is the most efficient for discovery of current values.
 To retrieve historical values, perform a log search using the `DSNPRegistryUpdate` event.
 
+### Current Handle -> Nonce
+
+The `IRegistry.resolveHandleToNonce` method is the only method for discovery of the current nonce.
```

Comment:
For me at least "current" meant most recently used (aka ethereum's getTransactionCount), but that is not what it means right?
```suggestion
The `IRegistry.resolveHandleToNonce` method is the only method for discovery of the next nonce to use.
```

Reply:
The current stored nonce should be next nonce to use (i.e. stored nonces should start at 0 and the first EIP 712 transaction should have a nonce of zero). I suppose the suggestion makes this a little more clear.